angular/src/app/workflows/workflows-routing.module.ts

```diff
@@ -1,12 +1,14 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
 
+import { WorkflowDetailComponent } from './workflow-detail/workflow-detail.component';
 import { WorkflowListComponent } from './workflow-list/workflow-list.component';
 import { WorkflowManageComponent } from './workflow-manage/workflow-manage.component';
 
 export const routes: Routes = [
   { path: '', component: WorkflowListComponent },
   { path: 'manage/add', pathMatch: 'full', component: WorkflowManageComponent },
+  { path: ':damId/views/:viewId/runs/:runId', component: WorkflowDetailComponent},
```

Comment:
I am not sure about this:
1. it is too long
2. `damId` and `viewId` are our internal variables for this request

It can be part of URL but it should be not part of URL path. The URL path should be only `/{realm}/workflows/{runId}`. See [matrix URL notation](https://angular.io/guide/router#navigating-back-to-the-list-component).

Reply:
On a second thought let's keep it as it is for now